### ISSUE TITLE
Using UC with asyncio and aiohttp

### ISSUE THREAD

**Issue by Altimis** [ultrafunkamsterdam/undetected-chromedriver#883]
Hi guys. Thanks for this brilliant library. Can we make undetected_chromedriver async using asyncio and aiohttp session please?

**Comment by sebdelsol** [comment]
You're right the driver is essentially IO ops, so that's a good candidate for asyncio. 

Please check [**this answer**](https://github.com/ultrafunkamsterdam/undetected-chromedriver/issues/451#issuecomment-1012378373) that uses a pretty nifty trick : browser tabs for parallelization.

The only issue with that is all driver explicit waits are blocking, so my guess is you'll need to check how [**arsenic**](https://github.com/HENNGE/arsenic) does it with an unmodified Selenium.

**Comment by DevilXD** [comment]
@Altimis I'm using this in my code to get around the driver being blocking:

```
loop = asyncio.get_running_loop()
driver = await loop.run_in_executor(None, lambda: us.Chrome(...))

await loop.run_in_executor(None, driver.get, "URL")
```

Sort of caveman code, and I don't have many interactions in my case (only really have these two), but they seem to be good enough to unblock the loop while still using this package. You can also write yourself a function that'd handle wrapping each call in an executor call like this, but I'm leaving that part to you.
